package.json

```diff
@@ -79,11 +80,16 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --write src/**/*.{js,jsx}",
+    "format": "prettier --write src/**/*.{js,jsx,scss}",
     "start": "npm run develop",
     "serve": "gatsby serve",
     "test": "jest",
     "storybook": "NODE_ENV=production start-storybook -s static -c config/storybook",
     "build-storybook": "NODE_ENV=production build-storybook -s static -c config/storybook"
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm run format && npm run test"
```

Comment:
We'll want to use `styled-lint` here so that the changes staged are added to the commit itself. I'll take full fault for this issue - I know I didn't lay out the requirements clearly enough before.